reference/test/matrix/hybrid_kernels.cpp

```diff
@@ -30,23 +30,23 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-#include "core/matrix/hybrid_kernels.hpp"
-
-
```

Comment:
I think this should stay since it is the header that is tested.

Reply:
For my original config, it should use <ginkgo/core/matrix/hybrid.hpp>.
I choose use <ginkgo/core/matrix/hybrid.hpp> not "core/matrix/hybrid_kernels.hpp" because it use apply to test and do not call the function directly.
If the script does not find the desired main header, it will ignore the main header part.

Reply:
It is fine with me if you add `ginkgo/core/matrix/hybrid.hpp` as the top include file.
I am not sure if we use any functions of that header, but to me, every test should have the corresponding header as the very first include file.